lib/plugins/place_block.js

```diff
@@ -22,7 +22,9 @@ function inject (bot, { version }) {
       dy = options.delta.y
       dz = options.delta.z
     }
-    await bot.lookAt(referenceBlock.position.offset(dx, dy, dz), false)
+    if (options.forceLook !== 'ignore') {
+      await bot.lookAt(referenceBlock.position.offset(dx, dy, dz), options.forceLook)
```

Comment:
You are passing here something of type string to something that expects a boolean

Reply:
Options.forceLook can be: 'ignore', false, true. If it is not 'ignore' - it is boolean. https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/digging.js#L30-L40 - here the same thing.

Reply:
Is it handled by the function you're passing it too ?

Reply:
No

Reply:
lookAt isnt called if forceLook isnt a boolean

Reply:
So i should add check? 
```
If (options.forceLook !== 'ignore' && typeof options.forceLook !== Boolean) {
    throw new Error('forceLook invalid input')
}
```
Or just in doc add info about types.